src/core/network.js

```diff
@@ -115,20 +130,22 @@ var NetworkManager = (function NetworkManagerClosure() {
         pendingRequest.expectedStatus = 200;
       }
 
-      if (args.onProgressiveData) {
-        // Some legacy browsers might throw an exception.
-        try {
-          xhr.responseType = 'moz-chunked-arraybuffer';
-        } catch(e) {}
-        if (xhr.responseType === 'moz-chunked-arraybuffer') {
-          pendingRequest.onProgressiveData = args.onProgressiveData;
-          pendingRequest.mozChunked = true;
-        } else {
-          xhr.responseType = 'arraybuffer';
-        }
+//#if GENERIC
+      if (args.onProgressiveData && supportsMozChunked) {
+//#endif
+//#if (FIREFOX || B2G || MOZCENTRAL)
+//    if (args.onProgressiveData) {
+//#endif
+//#if (GENERIC || FIREFOX || B2G || MOZCENTRAL)
+        xhr.responseType = 'moz-chunked-arraybuffer';
+        pendingRequest.onProgressiveData = args.onProgressiveData;
+        pendingRequest.mozChunked = true;
       } else {
         xhr.responseType = 'arraybuffer';
       }
+//#else
+      xhr.responseType = 'arraybuffer';
```

Comment:
Won't this line be an issue during development, i.e. when using `node make server`?
I believe that you simply need to comment-out this line to address that.